src/server/Document/Renderers/litePageTransforms/index.ts

```diff
@@ -0,0 +1,10 @@
+import transformAnchorTags from './transformAnchorTags';
+
+type Fn = (html: string) => string;
+
+const pipe =
```

Comment:
Do we plan to utilise this more widely long term in other areas of the app? i.e should we make it, it's own module/helper?

Reply:
Not sure how 1-2-1 it is against Ramdas `pipe` function (I'm sure theirs is much more involved!), but I just created this here so that I didn't need to use Ramda tbh.

Reply:
yeah no worries, looks cool!